packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -764,6 +780,16 @@ class FlutterDriver {
     }
   }
 
+  Future<bool> _isPrecompiledMode() async {
+    final List<Map<String, dynamic>> flags = await getVmFlags();
```

Comment:
How long does this take?  Would it be faster to do soemthing like:

```
bool assertsEnabled = false;
assert(() {
  assertsEnabled = true;
}());
return assertsEnabled;
```
?

Reply:
I tried that before exploring and fixing this buggy `getVmFlags` function (which was never used in our repo). Somehow, the code inside `assert` is never run. I thought maybe `flutter drive` itself is running the host dart code in non-debug mode, while the client dart code is running in debug mode.

Reply:
BTW, since this is run after the tracing is done, it shouldn't affect the tracing numbers. In terms of the test time cost, it probably adds no more than 0.5 second since I couldn't see any significant test slowdown.